source/css/_board.css

```diff
@@ -13,10 +19,11 @@
 }
 
 .board {
+  position: relative;
   width: var(--board-width);
+  max-width: 1000px;
   height: var(--board-width);
-  background: url('13x13.svg') no-repeat center transparent;
-  background-size: calc(var(--board-width) - var(--node-size));
+  max-height: 1000px;
```

Comment:
This is still not totally great for wider screens, but its more workable than before